apps/heft/build.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const path = require('path');
+const glob = require('glob');
+const { FileSystem } = require('@rushstack/node-core-library');
+const { ToolPaths } = require('@microsoft/rush-stack-compiler-3.7');
+const { callNodeScript } = require('./callNodeScript');
+const child_process = require('child_process');
+
+const isProductionBuild = process.argv.indexOf('--production') !== -1;
+
+process.exitCode = 1;
+try {
```

Comment:
@iclanton This is so exciting!

I'll probably have some more feedback about the architecture, but I hesitate to voice opinions until there's enough functionality to see it building a somewhat realistic project.  Like, maybe we can aim to replace this `build.js` file for Heft itself.  

Another option is the TSDoc repo, which has some real-world [build.js files](https://github.com/microsoft/tsdoc/blob/master/tsdoc/build.js) that are pretty primitive and probably easy candidates.

Reply:
Agreed on all points. The architecture definitely needs some revision, but it's a good start.